### ISSUE TITLE
Initializing Appboy causes warning in console

### ISSUE THREAD

**Issue by sberrevoets** [Appboy/appboy-ios-sdk#49]
Upon first initialization of Appboy we are getting the following warning in the console:

```
"*** -[NSKeyedUnarchiver initForReadingWithData:]: data is NULL"
```

This a (cleaned up) stack trace when this happens:

```
Appboy_iOS_SDK`-[ABKTriggerManager initWithPersistentData:inAppMessageController:appboyOptions:] at ABKTriggerManager.m:25
Appboy_iOS_SDK`-[Appboy initWithApiKey:withAppboyOptions:] at Appboy.m:197
Appboy_iOS_SDK`+[Appboy startWithApiKey:inApplication:withLaunchOptions:withAppboyOptions:] at Appboy.m:119
```

Could you please take a look at this? This is on version 2.19.0, and I don't think it was happening in 2.18.4. Let me know if you need more info.

**Comment by briancaw** [comment]
Thanks for reporting this @sberrevoets.  We've identified the issue (it's undesirable but it doesn't have any functional effects) and we should be able to update with a fix in the next 24 hours.  We'll update when the fix is live.  

Please let us know if you are seeing any functional issues or side effects beyond the logs.

Thanks again,
Brian

**Comment by sberrevoets** [comment]
No adverse side effects as far as I can tell, just the warning in the console. Thanks, look forward to your fix!

**Comment by briancaw** [comment]
Hi @sberrevoets,

After investigating a bit more, we found this log (and the functionality that triggered it) has no side effects at all beyond the logging that you saw.  When I initially reported the 24 hour release timeline I thought this was going to be spamming our servers with exception info, but that won't be the case (it's not an exception, just a log message from `NSKeyedUnarchiver`.

As a result, we'd prefer to hold off releasing the fix until the next planned release.  If you strongly prefer we release sooner, please let us know.

Thanks,
Brian

**Comment by sberrevoets** [comment]
Ideally it'd be fixed sooner rather than later so our console remains readable, but if you're planning another release for the next week or so (as your recent release history has been) I can live with that as well.

**Comment by Wenzhi** [comment]
Hi @sberrevoets,

This issue is fixed in the new SDK version 2.19.1.  If you could confirm this issue is resolved it would be greatly appreciated.

Thanks,
Wenzhi

**Comment by sberrevoets** [comment]
Hey @Wenzhi, it appears that the latest update has fixed the problem. Thanks!
